book/src/specs/kimchi.md

```diff
@@ -1661,6 +1731,10 @@ pub struct VerifierIndex<G: KimchiCurve> {
     /// Foreign field modulus
     pub foreign_field_modulus: Option<BigUint>,
 
+    /// Keccak rotation table
+    #[serde_as(as = "Option<[[o1_utils::serialization::SerdeAs; 5]; 5]>")]
+    pub keccak_rotation_table: Option<[[G::ScalarField; 5]; 5]>,
```

Comment:
It is indeed something related to the keccak gadget only. But since it is part of the definition of the algorithm, I was not sure if this needed to be somewhere else more "accessible" for the verifier to check or something.